Add .dockeringnore file

levidevpanel opened the pull request.

levidevpanel committed:
Add .dockeringnore file

darrenoh commented on .dockerignore:
```diff
@@ -0,0 +1,6 @@
+*.log
+.env
+Dockerfile.dev
+README.md
```
The README needs to be in the app. What is the effect of removing it from the image?
```suggestion
```

darrenoh reviewed (changes requested):
@copilot Apply requested changes.

Copilot committed:
fix: keep README and remove stale Dockerfile.dev from dockerignore

Agent-Logs-Url: https://github.com/drupalforge/docker_publish_action/sessions/49ac7bc5-3f0c-4fec-ac6d-7a7ef8d95d75

Co-authored-by: darrenoh <2293701+darrenoh@users.noreply.github.com>
